.jenkins/infrastructure/dockerfiles/linux/Dockerfile.full

```diff
@@ -78,3 +89,11 @@ RUN echo "${UNAME} ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 # Setup devkit
 RUN curl ${devkits_uri} | tar xvz --no-same-permissions --no-same-owner
 RUN echo ${devkits_uri##*/} > /devkits/TARBALL
+
+# Set up out-of-proc attestation
+RUN groupadd sgx_prv
+RUN usermod -aG sgx_prv ${UNAME}
+ENV SGX_AESM_ADDR=1
```

Comment:
These lines make developers environment safe from the SGX in-kernel transition. However, they also make the developers unaware of the fact that their application users have to do the same.